deployment/scripts/initialize-external-operator-identity.sh

```diff
@@ -12,11 +12,11 @@ fi
 
 EXTERNAL_CERT="externalOperatorCert"
 EXTERNAL_KEY="externalOperatorKey"
-OPERATOR_DIR="/operator"
-EXTERNAL_IDENTITY_DIR="${OPERATOR_DIR}/external-identity"
+DEPLOYMENT_DIR="/deployment"
+EXTERNAL_IDENTITY_DIR="${DEPLOYMENT_DIR}/external-identity"
 NAMESPACE=`cat /var/run/secrets/kubernetes.io/serviceaccount/namespace`
-OPERATOR_CONFIG_DIR=${OPERATOR_DIR}/config
-OPERATOR_SECRETS_DIR=${OPERATOR_DIR}/secrets
+OPERATOR_CONFIG_DIR=${DEPLOYMENT_DIR}/config
+OPERATOR_SECRETS_DIR=${DEPLOYMENT_DIR}/secrets
```

Comment:
This is more of a "just curious". I very much understand using "/deployment" to align the operator and the webhook... but why change the mount location for the CM and Secret?

Reply:
The location of cert and key file is using the `getDeploymentHome` method of the `CoreDelegate` in `InitializeInternalIdentityStep` . I thought it was consistent with keeping all the identity files included config and secret mounted cert and key under `/deployment` dir. 

```
    this.configInternalCertFile = new File(delegate.getDeploymentHome(), "/config/internalOperatorCert");
    this.secretsInternalKeyFile = new File(delegate.getDeploymentHome(), "/secrets/internalOperatorKey");  
```